md/bonita-docker-installation.md

```diff
@@ -61,10 +80,62 @@ See the official PostgreSQL documentation for more details.
 docker run --name bonita_postgres --link mydbpostgres:postgres -d -p 8080:8080 bonitasoft/bonita-subscription:${varVersion}.0
 ```
 
-#### PostgreSQL
-if you don't want to run your database in a docker container, some things need to be configured separately: 
+#### Using docker-compose
+
+Create a file stack.yml with the following content:
+```
+# Use tech_user/secret as user/password credentials
+version: '3'
+
+services:
+  db:
+    image: postgres:11.6
```

Comment:
image 11.6 is not yet available => 11.5 is the latest (as a docker image)

Reply:
I set 11.2 (as used previously)